plane_crash/map.conf

```
@@ -0,0 +1,13 @@
+team.2.color = blue
+team.2.pos = (54,-5,-11)
+team.1.pos = (3,1,96)
+team.1.color = red
+team.1 = red
+author = minetevc
+rotation = z
+h = 140
+team.2 = blue
+name = Plane Crash In Mid-Summer
+initial_stuff = default:pick:steel,default:torches:10
```

There is a bug in initial stuff.
```suggestion
initial_stuff = default:pick_steel,default_torch 10
```